Registry/application_lifecycle_events.setreg

```diff
@@ -24,7 +24,8 @@
                 "FileIOAvailable": {},
                 "FileIOUnavailable": {},
                 "LegacySystemInterfaceCreated": {},
-                "CriticalAssetsCompiled": {}
+                "CriticalAssetsCompiled": {},
+                "CommandLineProcessed": {}
```

Comment:
I would claim this should be called "LegacyCommandLine" processed, since it is sent after CrySystem has processed the Command Line.
The ComponentApplication processes the command line in it's [constructor](https://github.com/o3de/o3de/blob/development/Code/Framework/AzCore/AzCore/Component/ComponentApplication.cpp#L407-L408)